tracer/test/Datadog.Trace.Tests/Telemetry/Collectors/DependencyTelemetryCollectorTests.cs

```diff
@@ -41,17 +41,18 @@ public void HasChangesAfterAssemblyLoaded()
         }
 
         [Fact]
-        public void DoesNotHaveChangesWhenSameAssemblyAddedTwice()
+        public void DoesHaveChangesWhenSameAssemblyAddedTwice()
```

Comment:
Hmmm, I don't think we want this to change 🤔 We still don't want to record the assembly twice, otherwise we essentially are creating the same problem we already have. And this may well happen in practice, because we load multiple domains.

Reply:
I guess you're right. Reworked in d52f09aadf1850367dde0f77fedcca62515b0aa8